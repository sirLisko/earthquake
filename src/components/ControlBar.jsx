import React from 'react';

import Filter from 'components/Filter';
import ClearFilter from 'components/ClearFilter';

const ControlBar = () => (
  <div>
    <Filter name="mag" />
    <Filter name="magType" />
    <ClearFilter />
  </div>
);

export default ControlBar;
