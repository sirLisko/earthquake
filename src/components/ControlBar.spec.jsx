import React from 'react';
import { shallow } from 'enzyme';

import ControlBar from './ControlBar';

describe('ControlBar Component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ControlBar />);
  });

  it('should render properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
