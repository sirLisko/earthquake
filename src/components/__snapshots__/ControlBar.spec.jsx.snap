// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ControlBar Component should render properly 1`] = `
<div>
  <Filter
    name="mag"
  />
  <Filter
    name="magType"
  />
  <ClearFilter />
</div>
`;
